docker.sh

```diff
@@ -121,6 +121,7 @@ echo "# ----------------------------------------------------------------------"
 if [ "${DEPEND}" == "build" ]; then
     ${DEBUG} docker build \
         --pull \
+        --secret id=github_token,env=GITHUB_PAT \
```

Comment:
I'd never have guessed what this does without reading the docs: "Take the env var `GITHUB_PAT` from the environment where the build is running, and make its contents available to the build, but not to the built image, as a file whose path will default to `/run/secrets/github_token`"